test/README.md

```diff
@@ -16,18 +16,6 @@ make
 # or
 make edge_core
 ```
-
-### Install dependency
-
-You need install **mosquitto** to support mqtt.
-
-xref: https://mosquitto.org/download/
-
-```shell
-# For ubuntu
-apt install -y mosquitto
-```
-
 ### Configure
 
 #### Modify the configuration files accordingly
```

Comment:
this need not be a heading .

Reply:
the description could be more meaningful . describing why this step is needed...like beehive has modules , to introduce new module need to register it. this is the way to register etc.